test/core/host_api/offchain_extension_test.cpp

```diff
@@ -299,15 +298,14 @@ TEST_P(TernaryParametrizedTest, LocalStorageCAS) {
   WasmPointer expected_pointer = 45;
   WasmSize expected_size = 45;
   WasmSpan expected_span = PtrSize(expected_pointer, expected_size).combine();
-  auto expected = Buffer{scale::encode(std::optional<Buffer>{}).value()};
+  Buffer expected(8, 'e');
```

Comment:
Please revert this file (it's a fix from master)

Reply:
It will reverted automatically when temporary merged branch will be removed by rebase after merge that branch to master